### DISCUSSION TITLE
In which famous British university would you find the Bodleian Library?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#707] [Galaxy Trivia] [answered]
**Bonus:** Which famous movie (3 series) was filmed there? 🎬

**Comment by peterbaccaro** [comment]
The Bodleian Library is at the **University of Oxford**. 
It has been featured in several famous movies over the years, one of the most notable movies filmed there is "**Harry Potter and the Philosopher's Stone**" (released in the US as "**Harry Potter and the Sorcerer's Stone**"), which was the first film in the Harry Potter series.

**Comment by SANKARAMDAS** [comment]
The building stood at the heart of Oxford's 'academic quarter', close to the schools in which lectures were given. By 1488, the room was superseded by the library known as Duke Humfrey's, which constitutes the oldest part of the Bodleian.

**Comment by thuongtruong1009** [comment, marked as answer]
The Bodleian Library is located in the city of Oxford, England. It is part of the University of Oxford, one of the most prestigious and well-known universities in the world.

The Bodleian Library has been featured in the famous movie series "Harry Potter." Specifically, it was used as a filming location for various scenes in the Hogwarts Library in the "Harry Potter" film series. The library's grandeur and historical ambiance made it a perfect setting for the magical library depicted in the movies. The Bodleian Library appeared in multiple films within the "Harry Potter" series, including "Harry Potter and the Philosopher's Stone," "Harry Potter and the Chamber of Secrets," and "Harry Potter and the Goblet of Fire."
